### DISCUSSION TITLE
want specific styled type of Handle for Nodes

### DISCUSSION THREAD

**Discussion by RanitManik** [xyflow/xyflow#4823] [Q&A] [answered]
## Reality
![image](https://github.com/user-attachments/assets/1b3dde1a-3bd3-457d-a0ef-5226e39cdd6c)

## Expectation
![image](https://github.com/user-attachments/assets/4b662f24-f120-45ce-bd4f-41073b4144f7)

### how can i make sure that my nodes have this specific type of handles?

**Comment by chrtze** [comment, marked as answer]
You need to copy all the styles from the examples (including `xy-theme.css`).

**Reply by RanitManik** [reply]
### Thanks for the help
